extendeddatacrossword_test.go

```diff
@@ -156,11 +156,11 @@ func BenchmarkRepair(b *testing.B) {
 			// Randomly remove 1/2 of the shares of each row
 			for r := 0; r < i*2; r++ {
 				for c := 0; c < i; {
-					ind := rand.Intn(i + 1)
-					if flattened[r*i+ind] == nil {
+					ind := rand.Intn(i * 2)
```

Comment:
I'd recommend renaming `i` to something more meaningful, e.g. `origDataSize` and used `eds.Width()` or a new variable instead of `i*2` (e.g. extended width).

Reply:
Not a bad idea.

Reply:
Fixed in b0c81c7.